### DISCUSSION TITLE
How to access to Electron webContents? Or other electron APIs

### DISCUSSION THREAD

**Discussion by elfeffe** [NativePHP/laravel#166] [Q&A] [answered]
I don't get how to access the functions in the Electron API.
Any idea?

**Comment by simonhamp** [comment, marked as answer]
I don't believe you can currently. I think you'd inject the Electron API's to your Laravel app's front-end Javascript and we don't do that for you (it's _your_ front-end).

I'm sure there will be cases where that becomes useful, but the intention is to make the runtime's APIs (in this case Electron) available through a consistent interface, via the Laravel back-end.

Basically, whenever you're reaching to do something that's typically provided by Electron, if there isn't an interface in NativePHP for how to do it, that should become a [feature request](https://github.com/orgs/nativephp/discussions/new?category=ideas).
